docs/draft-alakuijala-brotli-07.nroff

```diff
@@ -1742,7 +1742,7 @@ The decoding algorithm that produces the uncompressed data is as follows:
       else
          fill CMAPD[] with zeros
       read array of prefix codes for literals, HTREEL[]
-      read array of prefix codes for insert-and-copy, HTREEI[]
+      read array of prefix codes for insert-and-copy lengths, HTREEI[]
```

Comment:
Since this is in a .nf .fi block, we have to make sure that the line length does not exceed 72. 

We can make it fit like this:
         read array of literal prefix codes, HTREEL[]
         read array of inset-and-copy length prefix codes, HTREEI[]
         read array of distance prefix codes, HTREED[]